### ISSUE TITLE
Metadata profile discussion

### ISSUE THREAD

**Issue by RaduGiurgiu** [soilwise-he/Project-management#32]

**Comment by dluts** [comment]
Is there some kind of example or document describing the proposed profile available?

**Comment by pvgenuchten** [comment]
Yes, the profile has been added at https://github.com/soilwise-he/soilwise-ontology/blob/main/docs/metadata.md

**Comment by RaduGiurgiu** [comment]
There is planned a discussion with REA and JRC on 11.09 on Metadata structure. @pvgenuchten can you organize an internal meeting before that to discuss it from SoilWise's pov?
